modules/core/04-channel/keeper/timeout_test.go

```diff
@@ -268,6 +270,7 @@ func (suite *KeeperTestSuite) TestTimeoutExecuted() {
 				channel := path.EndpointA.GetChannel()
 				suite.Require().Equal(channel.State, types.CLOSED)
 			},
+			nil,
```

Comment:
i think this case might actually have an event attached to it -- it might be nicer for the test if you run it [this way](https://github.com/cosmos/ibc-go/blob/eaabb85c67adbefb53cd32aecd1ae9680d7e9e9a/modules/core/04-channel/keeper/upgrade_test.go#L136C24-L136C24) so we can avoid missing cases

Reply:
or, just run this test on all success cases

Reply:
actually! nevermind this case is not missing, it is ORDERED!

Reply:
oh got it, thanks for the review thou